guice/src/test/java/io/cucumber/guice/GuiceFactoryTest.java

```diff
@@ -58,9 +58,9 @@ void factoryStartFailsIfScenarioScopeIsNotBound() {
         Executable testMethod = () -> factory.start();
         ConfigurationException actualThrown = assertThrows(ConfigurationException.class, testMethod);
         assertThat("Unexpected exception message", actualThrown.getMessage(), is(equalTo(
-            "Guice configuration errors:\n\n" +
-                "1) No implementation for io.cucumber.guice.ScenarioScope was bound.\n" +
-                "  while locating io.cucumber.guice.ScenarioScope\n\n" +
+            "Guice configuration errors:" + System.lineSeparator() + System.lineSeparator() +
+                "1) No implementation for io.cucumber.guice.ScenarioScope was bound." + System.lineSeparator() +
+                "  while locating io.cucumber.guice.ScenarioScope" + System.lineSeparator() + System.lineSeparator() +
```

Comment:
It might better to use `equalToCompressingWhiteSpace`instead. It is allot more readable. Both here and in `DocStringTypeRegistryDocStringConverterTest`.

Reply:
Nice. I've updated code.